data-hibernate-jpa/src/main/java/io/micronaut/data/hibernate/naming/DefaultPhysicalNamingStrategy.java

```diff
@@ -43,12 +46,15 @@ public Identifier toPhysicalSchemaName(Identifier name, JdbcEnvironment jdbcEnvi
 
     @Override
     public Identifier toPhysicalTableName(Identifier name, JdbcEnvironment jdbcEnvironment) {
+        if (isSequence(name)) {
```

Comment:
This is where sequence is being created, expected in `toPhysicalSequenceName` but this is being called